src/main/java/frc/robot/subsystems/Limelight.java

```diff
@@ -0,0 +1,155 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+
+public class Limelight extends SubsystemBase {
+    private NetworkTableInstance ntInstance = NetworkTableInstance.getDefault();
+    private NetworkTable table = ntInstance.getTable("/limelight");
+
+    private NetworkTableEntry yaw = table.getEntry("/tx");
+    private NetworkTableEntry pitch = table.getEntry("/ty");
+    private NetworkTableEntry lightMode = table.getEntry("/ledMode");
+    private NetworkTableEntry validTarget = table.getEntry("/tv");
+    private NetworkTableEntry skew = table.getEntry("/ts");
+    private NetworkTableEntry cameraMode = table.getEntry("/camMode");
+    private NetworkTableEntry pipeline = table.getEntry("/pipeline");
+    private NetworkTableEntry targetState = table.getEntry("/target_state");
+    private NetworkTableEntry poseData = table.getEntry("/camtran");
+
+    public static class Constants {
+        // LED MODES /limelight/ledMode
+        public static final int LED_MODE_FROM_PIPELINE = 0;
+        public static final int LED_MODE_FORCE_OFF = 1;
+        public static final int LED_MODE_FORCE_BLINK = 2;
+        public static final int LED_MODE_FORCE_ON = 3;
```

Comment:
Maybe use `enum` class for these groups of constants? Not required but then you can force the usage by using the enum class as the variable type